packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel.Plugin/src/Microsoft.Generator.CSharp.ClientModel.Plugin.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Generator.CSharp.ClientModel.Plugin</PackageId>
```

Comment:
is this another test plugin?  If so lets name something without the mgc since its mimicing a 3p plugin?

Reply:
No this is the SCM plugin project.